src/components/MagPage/styles/magazinePage.css

```diff
@@ -0,0 +1,31 @@
+.bkgImage {
```

Comment:
It looks like we've gone in the direction of CSS files that exactly patch the name of the component as a style for this project, and use first letter capitalization like the component files.. Can you change the name of this file?